source/OutfitterPanel.cpp

```diff
@@ -101,7 +101,8 @@ void OutfitterPanel::Step()
 	if(GetUI()->IsTop(this) && !checkedHelp)
 		// Use short-circuiting to only display one of them at a time.
 		// (The first valid condition encountered will make us skip the others.)
-		if(DoHelp("outfitter") || DoHelp("cargo management") || DoHelp("uninstalling and storage") || true)
+		if(DoHelp("outfitter") || DoHelp("cargo management") || DoHelp("uninstalling and storage")
+				|| (player.Ships().size() > 1 && DoHelp("outfitter with multiple ships")) || true)
```

Comment:
Would it not make more sense to only show this message if the player has more than one ship *here*?